Dockerfile

```diff
@@ -16,7 +16,8 @@ RUN find /www \
   -exec gzip -9 -k '{}' \;
 
 RUN chown -R nginx:nginx /www
-RUN chmod -R u=rwx /www
+RUN chmod u=rwx /www
```

Comment:
Changed this to apply `rwx` permissions only to the directory, not the files in it. IMO there is no point in an executable `index.html`